src/coordination/coordinator_instance.cpp

```diff
@@ -72,11 +74,24 @@ CoordinatorInstance::CoordinatorInstance(CoordinatorInstanceInitConfig const &co
                   repl_instance_name);
     std::invoke(repl_instance.GetFailCallback(), self, repl_instance_name);
   };
+  auto const coordinator_state_manager_durability_dir = config.durability_dir / "state_manager";
+  memgraph::utils::EnsureDirOrDie(coordinator_state_manager_durability_dir);
+  CoordinatorStateMachineConfig state_machine_config{config.coordinator_id, nullptr};
+  CoordinatorStateManagerConfig state_manager_config{config.coordinator_id, config.coordinator_port, config.bolt_port,
+                                                     coordinator_state_manager_durability_dir, nullptr};
+
+  if (FLAGS_coordinator_use_durability) {
```

Comment:
You should receive this info from the config object in constructor, don't directly use flags